### PR TITLE
[Fluid Clean] remove apis in fluid.layers.ops

### PR DESCRIPTION
**Pull request by zoooo0820**
### PR types

Others
### PR changes

APIs
### Describe

Remove apis in fluid.layers.ops, including these apis:
```
paddle.fluid.layers.sigmoid
paddle.fluid.layers.silu
paddle.fluid.layers.logsigmoid
paddle.fluid.layers.tanh_shrink
paddle.fluid.layers.softsign
paddle.fluid.layers.tanh
paddle.fluid.layers.exp
paddle.fluid.layers.expm1
paddle.fluid.layers.atan
paddle.fluid.layers.sqrt
paddle.fluid.layers.rsqrt
paddle.fluid.layers.abs
paddle.fluid.layers.ceil
paddle.fluid.layers.floor
paddle.fluid.layers.cos
paddle.fluid.layers.tan
paddle.fluid.layers.acos
paddle.fluid.layers.sin
paddle.fluid.layers.sinh
paddle.fluid.layers.asin
paddle.fluid.layers.cosh
paddle.fluid.layers.round
paddle.fluid.layers.reciprocal
paddle.fluid.layers.square
paddle.fluid.layers.acosh
paddle.fluid.layers.asinh
paddle.fluid.layers.atanh
paddle.fluid.layers.lgamma
paddle.fluid.layers.exp_
paddle.fluid.layers.sqrt_
paddle.fluid.layers.rsqrt_
paddle.fluid.layers.ceil_
paddle.fluid.layers.floor_
paddle.fluid.layers.round_
paddle.fluid.layers.reciprocal_
paddle.fluid.layers.softplus
paddle.fluid.layers.softshrink
paddle.fluid.layers.hard_shrink
paddle.fluid.layers.cumsum
paddle.fluid.layers.threshold_relu
paddle.fluid.layers.gelu
paddle.fluid.layers.erf
paddle.fluid.layers.ops._elementwise_div
```

All of above apis have a same implement in paddle.tensor.ops, so we can remove them directly.

### CODE REVIEW

**Review by jeff41404 — DISMISSED** [review]

**Review by dingjiaweiww — DISMISSED** [review]
Looks good

**Review by dingjiaweiww — DISMISSED** [review]
Looks good

**Review by jeff41404 — DISMISSED** [review]

**Review by jeff41404 — DISMISSED** [review]

**Review by dingjiaweiww — DISMISSED** [review]

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py [hunk 1]
```diff
@@ -601,15 +601,15 @@ def __init__(
         self.epochs = epochs
 
     def step(self):
-        from .. import layers
+        import paddle
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/learning_rate_scheduler.py]: same

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py [hunk 2]
```diff
@@ -437,11 +437,11 @@ def __init__(
         self.staircase = staircase
 
     def step(self):
-        from .. import layers
+        import paddle
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/learning_rate_scheduler.py]: same

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py [hunk 3]
```diff
@@ -356,11 +356,11 @@ def __init__(
...
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/learning_rate_scheduler.py]: same

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py [hunk 4]
```diff
@@ -270,12 +270,12 @@ def __init__(
...
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/learning_rate_scheduler.py]: why this? this may cause performance issue

**Review by JiabinYang — COMMENTED** [review]
some comments

File: python/paddle/fluid/dygraph/learning_rate_scheduler.py (hunk 4 above)
Comment (zoooo0820) [reply python/paddle/fluid/dygraph/learning_rate_scheduler.py]: Done, already remove the code in the file.

**Review by JiabinYang — APPROVED** [review]
LGTM

**Review by dingjiaweiww — APPROVED** [review]

**Review by jeff41404 — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**